src/main/java/BuildingCostCalculator.java

```diff
@@ -0,0 +1,68 @@
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Scanner;
+
+
+public class BuildingCostCalculator {
+
+    private static final Logger logger = LogManager.getLogger(Main.class);
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        logger.info("======================================");
+        logger.info("|         Building Cost Calculator     |");
+        logger.info("======================================");
+
+        System.out.print("Enter the building type: ");
+        String buildingType = scanner.nextLine();
+
+        System.out.print("Enter the floor area (in square meters): ");
+        double floorArea = scanner.nextDouble();
```

Comment:
The user can enter a different type of data. You need to add exception handling.